Sources/Sentry/SentryScope.m

```diff
@@ -94,6 +97,7 @@ - (instancetype)initWithScope:(SentryScope *)scope
         self.environmentString = scope.environmentString;
         self.levelEnum = scope.levelEnum;
         self.fingerprintArray = scope.fingerprintArray.mutableCopy;
+        self.attachmentArray = scope.attachmentArray.mutableCopy;
```

Comment:
`attachmentArray` is public on the scope? We're accessing a mutable object of a foreign object outside of any synchronization.

unless `attachmentArray` is a property that does `@syncronize` under the hood